### ISSUE TITLE
Consultas complexas

### ISSUE THREAD

**Issue by braganholo** [gems-uff/sapos#10]
- Permitir consultas complexas (por exemplo, cruzar aluno com bolsa, aluno com orientador, etc.), e deixar que o usuário possa criar outras consultas sempre que necessário, salvando-as, sem ter que pedir para os desenvolvedores implementarem. A consulta poderia ser escrita em SQL via formulário, e salvas com um nome específico.
- Teria como rodar as consultas com outro usuário do BD? Isso permitiria criarmos um usuário no BD com acesso só de leitura, para evitar efeitos colaterais.
- Teria como permitir que, ao escrever a consulta, fossem criadas variáveis? Assim, ao escolher uma consulta para rodar, o usuário poderia preencher parâmetros que seriam atribuídos a essas variáveis.
- Seria interessante fornecer uma possibilidade de gerar o resultado da consulta em pdf, para facilitar impressão e envio em meio digital.

**Comment by leomurta** [comment]
Disso aqui, parece que tudo foi feito, menos a geração de pdf, certo @braganholo ? Na verdade, eu acho que seria mais útil gerar Excel, pois na prátia nós copiamos o resultado e colamos no Excel.

**Comment by braganholo** [comment]
Sim, está tudo ok, exceto a geração do PDF. Concordo que seria mais útil gerarmos um Excel (CSV para fins de compatibilidade).
